setup.sh

```diff
@@ -185,16 +203,34 @@ then
 		fi
 		info "Installing dependencies..."
 		sudo pacman -S --noconfirm --needed $ARCHDEBS >>$OUTFILE 2>>$ERRFILE
+	elif [ $IS_MACOS -eq 1 ]
+	then
+		if ! which brew > /dev/null;
+		then
+			error "Your system doesn't have homebrew installed, I don't know how to install the dependencies.\nPlease install homebrew: https://brew.sh/\nOr install the equivalent of these homebrew packages: $HOMEBREW_DEBS."
+		fi
+		brew install $HOMEBREW_DEBS >>$OUTFILE 2>>$ERRFILE
 	else
 		error "We don't know which dependencies to install for this sytem.\nPlease install the equivalents of these debian packages: $DEBS."
 	fi
 fi
 
 info "Checking dependencies..."
-[ -e /etc/debian_version ] && [ $(dpkg --get-selections $DEBS | wc -l) -ne $(echo $DEBS | wc -w) ] && error "Please install the following packages: $DEBS" && exit 1
-[ -e /etc/pacman.conf ] && [ $(pacman -Qi $ARCHDEBS  2>&1 | grep "was not found" | wc -l) -ne 0 ] && error "Please install the following packages: $ARCHDEBS" && exit 1
-[ -e /etc/pacman.conf ] && [ $(pacman -Qi $ARCHCOMDEBS  2>&1 | grep "was not found" | wc -l) -ne 0 ] && error "Please install the following packages from AUR (yaourt -S <package_name>)): $ARCHCOMDEBS" && exit 1
-[ ! -e /etc/debian_version ] && [ ! -e /etc/pacman.conf ] && warning -e "WARNING: make sure you have dependencies installed.\nThe debian equivalents are: $DEBS.\nPress enter to continue." && read a
+if [ -e /etc/debian_version ]
+then
+	[ $(dpkg --get-selections $DEBS | wc -l) -ne $(echo $DEBS | wc -w) ] && error "Please install the following packages: $DEBS"
+elif [ -e /etc/pacman.conf ]
```

Comment:
this isn't if-elif - there's two arch clauses that both need to run. you could make it into if-elif by merging them, though

Reply:
Ohhhh, I see, thanks 👍

Reply:
Collapsed the arch statements into one.